/**:
  ros__parameters:
    color_width: 640
    color_height: 480
    color_fps: 30.0
    color_format: "RGB888"
    color_frame_id: "color_frame"
    color_optical_frame_id: "color_optical_frame"
    enable_color: true
    ir_width: 640
    ir_height: 480
    ir_fps: 30.0
    ir_format: "Y16"
    ir_frame_id: "ir_frame"
    ir_optical_frame_id: "ir_optical_frame"
    enable_ir: true
    depth_width: 1280
    depth_height: 1024
    depth_fps: 30.0
    depth_format: "Y16"
    depth_frame_id: "depth_frame"
    depth_optical_frame_id: "depth_optical_frame"
    enable_depth: true
    publish_tf: true
    tf_publish_rate: 10.0
    publish_rgb_point_cloud: false
    wait_for_device_timeout: 120.0
    reconnect_timeout: 6.0
    d2c_mode: "none"
    serial_number: ""
    camera_link_frame_id: "camera_link"
    ob_log_level: "none"
